### DISCUSSION TITLE
Ideas and Recommendations for new SCB Vulnerability Scanner

### DISCUSSION THREAD

**Discussion by rfelber** [secureCodeBox/secureCodeBox#1047] [Ideas]
Hi folks 👋

i would like to start a discussion about ideas, wishes and recommendations for new security vulnerability scanner which should be integrated into the secureCodeBox.

If you have any opinion on that, just drop a line here or vote for already existing suggestions 👍🏻

To make a first start i added some curated lists to get you inspired:
* https://github.com/sbilly/awesome-security
* https://github.com/qazbnm456/awesome-web-security
* https://github.com/rustcohlnikov/awesome-frontend-security
* https://github.com/aubrel/awesome-devsecops
* https://github.com/nahberry/awesome-recon-tools
* https://github.com/enaqx/awesome-pentest
* https://github.com/theepiccode/Awesome-CyberSec-Resources
* https://github.com/blaCCkHatHacEEkr/PENTESTING-BIBLE
* https://medium.com/@johnnyluther96/penetration-testing-hacking-tools-list-for-hackers-e1eb4e4cc507

Vulnerable Apps:
* https://github.com/kaiiyer/awesome-vulnerable
* https://github.com/vavkamil/awesome-vulnerable-apps

**Comment by Weltraumschaf** [comment]
I like the idea [scanning for Samba](https://github.com/ShawnDEvans/smbmap).

**Comment by J12934** [comment]
Another good list at: https://github.com/psiinon/open-source-web-scanners

**Comment by wurstbrot** [comment]
I didn't perform an evaluation, the following seems to be good addations:
https://github.com/armosec/kubescape
https://www.arachni-scanner.com/ (there is a new version since March 2022)
The auth checks looks interesting: https://github.com/blst-security/cherrybomb

**Comment by SebieF** [comment]
Didn't evaluate it but was for a long time on my to-try list:
https://github.com/FortyNorthSecurity/EyeWitness

Might replace the screenshooter.
